Update nf-synchapi-sleepex.md

axelriet opened the pull request:
Added a small precision, after reading %SDXROOT%\minkernel\ntos\ke\wait.c around lines 88-96: the code checks for the timeout being zero _and_ the alertable flag being false to decide to take the shortcut or not.

axelriet committed:
Update nf-synchapi-sleepex.md

Added a small precision, after reading %SDXROOT%\minkernel\ntos\ke\wait.c around lines 88-96: the code checks for the timeout being zero _and_ the alertable flag being false to decide to take the shortcut or not.

PRMerger18 commented:
@axelriet : Thanks for your contribution! The author(s) have been notified to review your proposed change.

axelriet committed:
Update nf-synchapi-sleepex.md

Added one more tidbit about the required absence of user APCs in order to quickly yield execution as described: three conditions are required: timeout = 0, alertable = false, and no user APCs are queued.

Karl-Bridge-Microsoft reviewed (approved):
Thanks @axelriet!

Karl-Bridge-Microsoft commented:
#sign-off
